### ISSUE TITLE
pixman not building for VS2017 in AppVeyor 

### ISSUE THREAD

**Issue by ofTheo** [openframeworks/apothecary#144]
I just installed VS2017 as per the OF setup instructions and manually installed git, make, cmake etc. 

Using the MINGW64 bash and  the same command:
`./apothecary -tvs -a64 -f -p update pixman`

It builds correctly. 

```
$ ./apothecary -tvs -a64 -f -p update pixman
OF: /c/Users/oftheo/Documents/GitHub/apothecary/apothecary/../../../
APOTHECARY: /c/Users/oftheo/Documents/GitHub/apothecary/apothecary
FORMULAS: /c/Users/oftheo/Documents/GitHub/apothecary/apothecary/formulas
BUILD: /c/Users/oftheo/Documents/GitHub/apothecary/apothecary/build
LIBS_DIR: /c/Users/oftheo/Documents/GitHub/apothecary/apothecary/../
ADDONS: /c/Users/oftheo/Documents/GitHub/apothecary/apothecary/../../../addons
REL_ADDONS_DIR: /c/Users/oftheo/Documents/GitHub/apothecary/apothecary/../../../addons
VS_VER was not set - setting to 15
PLATFORM set to 'x64', x64_15
Confirmed VS_BIN_PATH element exists: '/c/Program Files (x86)/Microsoft Visual Studio/2017/Community/VC/Tools/MSVC/14.16.27023/bin/HostX64/x64'
VS_BIN_PATH was not set - setting to /c/Program Files (x86)/Microsoft Visual Studio/2017/Community/VC/Tools/MSVC/14.16.27023/bin/HostX64/x64

 ----- pixman -----

 Downloading "pixman"

removing already existing pixman
2019-10-04 18:13:18 URL:https://www.cairographics.org/releases/pixman-0.32.4.tar.gz [817407/817407] -> "pixman-0.32.4.tar.gz" [1]

 Preparing "pixman"

 Building "pixman"

 Copying "pixman"

'pixman/release/pixman-1.lib' -> '/c/Users/oftheo/Documents/GitHub/apothecary/apothecary/..//pixman/../cairo/lib/vs/x64/pixman-1.lib'

 Finished "pixman"

``` 

For comparison this is what the AppVeyor error is:
https://ci.appveyor.com/project/arturoc/apothecary/builds/27433728/job/yfj5bwetw9y86896 

I did get a similar error as in AppVeyor before I installed `make` but I doubt that is the issue as I would imagine that is being used in other places? 

I wonder if its a AppVeyor configuration issue though?  
As it basically ( and the rest of the Cairo formulas ) builds fine on Desktop.

**Comment by ofTheo** [comment]
hmm - now looking at AppVeyor all builds are failing 🤦‍♂️
https://ci.appveyor.com/project/arturoc/apothecary/builds/27887991

**Comment by tgfrerer** [comment]
i think this issue has been solved with #148

**Comment by arturoc** [comment]
yes, thanks!!
